homeassistant/components/amcrest/camera.py

```diff
@@ -51,6 +52,43 @@
 _SRV_TOUR_ON = "start_tour"
 _SRV_TOUR_OFF = "stop_tour"
 
+_SRV_PTZ_CTRL = "ptz_control"
+_ATTR_PTZ_TT = "travel_time"
+_ATTR_PTZ_MOV = "movement"
+_MOV = [
+    "zoom_in",
+    "zoom_out",
+    "right",
+    "left",
+    "up",
+    "down",
+    "right_down",
+    "right_up",
+    "left_down",
+    "left_up",
+    "bottom_right",
+    "top_right",
+    "bottom_left",
+    "top_left",
+]
+_ACTION = [
+    "ZoomTel",
```

Comment:
nice typo catch!    I don't have an optical zoom camera, so zoom unfortunately wasn't tested.   Do you?

Reply:
I honestly don't remember if the PTZ camera I have does optical zoom, but I know it supports zooming (either optically or digitally), so I would guess it would support zooming this way. I'll give this stuff a try after this first review round and you check in the requested changes.